torch/csrc/api/include/torch/data/datasets/chunk.h

```diff
@@ -246,18 +259,18 @@ class BatchDataBuffer {
 struct ChunkDatasetOptions {
   ChunkDatasetOptions() = delete;
   ChunkDatasetOptions(
-      size_t preloader_count,
       size_t batch_size,
+      size_t preloader_count = 2,
       size_t cache_size = 2048)
-      : preloader_count_(preloader_count),
-        batch_size_(batch_size),
+      : batch_size_(batch_size),
```

Comment:
Just curious, is there a specific reason we want to switch the order?

Reply:
I think only batch_size is enough to create a chunk data set, the preloader_count and cache_size can have default values.

Reply:
Make sense.